src/frequenz/client/dispatch/_internal_types.py

```diff
@@ -42,8 +44,11 @@ class DispatchCreateRequest:
     start_time: datetime
     """The start time of the dispatch in UTC."""
 
-    duration: timedelta
-    """The duration of the dispatch, represented as a timedelta."""
+    duration: timedelta | None
+    """The duration of the dispatch, represented as a timedelta.
+
+    If None, the dispatch is considered to be "infinite" or "no-duration".
```

Comment:
For me "no-duration" is not super clear, I think I would say "instantaneous" (or maybe both), or use an example as you did in the docs above.